Sources/ARSAL_Mutex.c

```diff
@@ -145,11 +174,36 @@ int ARSAL_Mutex_Unlock(ARSAL_Mutex_t *mutex)
 #endif
             );
     }
+#elif defined(HAVE_WINDOWS_H)
+    result = ReleaseMutex((HANDLE)*mutex);
+    if (result != TRUE) {
+        int err = GetLastError();
+        ARSAL_PRINT(ARSAL_PRINT_FATAL, ARSAL_MUTEX_TAG,
+                    "Mutex unlock operation failed! GetLastError() = %d", err);
+        result = 1;
+    } else {
+        result = 0;
+    }
 #endif
 
     return result;
 }
 
+#ifdef HAVE_WINDOWS_H
```

Comment:
Indeed they are - but they do not work with mutexes, only critical sections or R/W locks, so I decided to just copy an implementation that works with mutexes.